apps/web/src/app/[locale]/challenge/[slug]/solutions/_components/index.tsx

```diff
@@ -69,7 +69,6 @@ export function Solutions({ slug }: Props) {
     queryKey,
     queryFn: () =>
       getPaginatedSolutions({ slug, page, sortKey: sortKey.key, sortOrder: sortKey.order }),
-    keepPreviousData: true,
```

Comment:
is this not a thing anymore?

Reply:
Looks like I missed it https://tanstack.com/query/latest/docs/react/guides/migrating-to-v5#removed-keeppreviousdata-in-favor-of-placeholderdata-identity-function